### DISCUSSION TITLE
希望添加一个进制转浮点的功能

### DISCUSSION THREAD

**Discussion by MrAccle** [he3-app/he3-feedback#91] [Ideas]
例如：
Hex -> IEEE 754浮点数
Float -> Hex

参考网站
http://www.speedfly.cn/tools/hexconvert/
![image](https://user-images.githubusercontent.com/25242380/228105937-41409df0-169f-45d7-b428-344a0e052ee9.png)

**Comment by MrAccle** [comment]
目前仅有的功能如下：
![image](https://user-images.githubusercontent.com/25242380/228106084-ffdd8ba0-0487-4c0a-8cb4-f2b07d50e173.png)

**Comment by lyzhang1999** [comment]
感谢反馈，已经创建了 issues：https://github.com/he3-app/he3-feedback/issues/103，将在后续加入到工具箱中。
